CHANGELOG.md

```diff
@@ -5,11 +5,19 @@ follows a format inspired by [Keep a Changelog](https://keepachangelog.com/en/1.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/panorama-ed/memo_wise/compare/v1.9.0...HEAD)
+## [Unreleased](https://github.com/panorama-ed/memo_wise/compare/v1.10.0...HEAD)
 
 **Gem enhancements:**
 
-- Reduced unzipped gem size from 173KB to 19KB [[#345](https://github.com/panorama-ed/memo_wise/pull/345)]
+_No breaking changes!_
+
+**Project enhancements:**
+
+## [v1.10.0](https://github.com/panorama-ed/memo_wise/compare/v1.9.0...v1.10.0)
+
+**Gem enhancements:**
+
+- Reduced unzipped gem size from 312 kB to 68 kB [[#345](https://github.com/panorama-ed/memo_wise/pull/345)]
```

Comment:
Switched from `KB` to `kB` because [Wikipedia told me](https://en.wikipedia.org/wiki/Kilobyte) `kB` is correct.